build.gradle

```diff
@@ -25,7 +27,17 @@ dependencies {
 
 intellij {
     version ideaVersion
+    type System.getenv().getOrDefault("INTELLIJ_TYPE", "IC")
```

Comment:
I just added this so that way I could run the Ultimate version of intellij when I develop. Variants that I know exist are `IC` and `IU` for community and ultimate respectively